pkg/trim-after-delete/main.go

```diff
@@ -136,7 +136,13 @@ RECONNECT:
 				continue RECONNECT
 			}
 			if event.Action == "delete" && event.Type == "image" {
-				log.Printf("The delayed action will happen at least once more")
+				log.Printf("An image has been removed: will run the action at least once more")
+				action.AtLeastOnceMore()
+			} else if event.Action == "destroy" && event.Type == "container" {
+				log.Printf("A container has been removed: will run the action at least once more")
+				action.AtLeastOnceMore()
+			} else if event.Action == "destroy" && event.Type == "volume" {
+				log.Printf("A volume has been removed: will run the action at least once more")
```

Comment:
Wondering if this will become very noisy (perhaps not a problem, just curious if every even should be logged here, or logging should also be delayed (or only in debug mode)

Reply:
I've not noticed whether it's a problem in practice or not... but I've not been looking. I'll keep an eye out!